Lib/functools.py

```diff
@@ -933,6 +933,7 @@ def __init__(self, func):
 
         self.dispatcher = singledispatch(func)
         self.func = func
+        self._dispatch_method = None
```

Comment:
I may be missing something, but nothing assigns `_dispatch_method`

Reply:
:blush: I added the missing assignment...